src/script.js

```diff
@@ -74,4 +87,14 @@ addButton.addEventListener('click', () => {
     new Item(inputValue.value);
     inputValue.value = '';
   }
-});
\ No newline at end of file
+});
+
+clearButton.addEventListener('click', () => {
+  itemList = itemList.filter((item) => !item.completed);
+  container.innerHTML = '';
+  itemList.forEach((item) => {
+    // eslint-disable-next-line no-new
+    new Item(item.itemName, item.completed);
+  });
+  localStorage.setItem('itemList', JSON.stringify(itemList));
+});
```

Comment:
- [ ] As the  [ the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md)[Check the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md), you should ensure that the tasks cleared are saved to the local storage. However, as shown in the video, the first tasks are deleted, making a copy of the first task that can not be cleared. Kindly ensure that this is fixed to ensure the clear functionality is working as intended ☺️.

Video presentation 😎.
https://clipchamp.com/watch/6HWKgxBzGqM